.gitmodules

```diff
@@ -1010,6 +1010,7 @@
 [submodule "vendor/grammars/vscode-codeql"]
 	path = vendor/grammars/vscode-codeql
 	url = https://github.com/github/vscode-codeql
+	branch = main
```

Comment:
This repo has done the `master` -> `main` rename dance.

Reply:
It's funny, I always thought `master` was a reference to the "master" disc of an audio recording (since branching off master implies a copy, similar to how phonographic discs are pressed).

Would have stayed that way, too, had people not drawn attention to it...